### PR TITLE
Allow polling timeout to be configurable

### PR DESCRIPTION
**Pull request by cblecker**

### CODE REVIEW

File: docs/Options.md [hunk 1]
```diff
@@ -40,6 +40,13 @@ These options are required to run osde2e.
 
 - Type: `string`
 
+### `POLLING_TIMEOUT`
+
+- PollingTimeout is how long to wait for an object to be created before
```
Comment (jeefy) [inline docs/Options.md]: Small discussion point: Is this for "objects" or for-specifically operators? Should we rename this completely to be operator-specific on the likelihood we will have additional objects we'll want to create-or-timeout with?

File: pkg/config/load.go [hunk 2]
```diff
@@ -15,6 +15,23 @@ func (c *Config) LoadFromEnv() {
 	v := reflect.ValueOf(c).Elem()
 	for i := 0; i < v.Type().NumField(); i++ {
 		f := v.Type().Field(i)
+		if defaultValue, ok := f.Tag.Lookup(DefaultTag); ok {
```
Comment (jeefy) [inline pkg/config/load.go]: We actually have a couple areas where we do if-nil-hard-coded-default. I like this better and I'll have a follow-up PR to update those.

File: docs/Options.md (hunk 1 above)
Comment (cblecker) [reply docs/Options.md]: Right now it's only being used for operators. I'm happy to rename if there is a better suggestion.

Comment (jeefy) [reply docs/Options.md]: It might warrant renaming later, for now I think it's reasonable. It was mainly a point for discussion. Thoughts @meowfaceman?

**Pull request merged**